apps/system/lockscreen/js/lockscreen_notifications.js

```diff
@@ -295,8 +299,12 @@
           node,
           oldLockScreenNode
         );
-      }
-      else {
+      } else if ((this.container.firstElementChild) && (this.container.firstElementChild.id === node.id)) {
+        this.container.replaceChild(
+          node,
+          this.container.firstElementChild
+        );
```

Comment:
If you really need this separate else clause in addition to the if clause above, please add a comment explaining why it is needed, and why it is a special case for the first element chlid. I don't understand this, and suspect it is wrong.